requirements.txt

```diff
@@ -10,6 +10,8 @@ djmail==0.11.0
 futures==3.0.3
 gunicorn==19.7.1
 lxml==4.1.0
+mock==2.0.0
```

Comment:
Where are you using this?

Reply:
It is needed in several tests including:
- hc/api/tests/test_notify.py
- hc/api/tests/test_sendalerts.py
- hc/front/tests/test_add_pushbullet.py
- hc/front/tests/test_add_slack.py
- hc/payments/tests/test_billing.py
- hc/payments/tests/test_cancel_plan.py
- hc/payments/tests/test_create_plan.py
- hc/payments/tests/test_get_client_token.py
- hc/payments/tests/test_invoice.py